.gitignore

```diff
@@ -202,4 +202,7 @@ fabric.properties
 # Google services
 app/google-services.json
 
+#Images
+app/src/main/res/drawable/teckerslogo.png
```

Comment:
Why are we ignoring the logo?

Reply:
Because you told me that i had to delete the bin of the logo.